pkg/spec/tf_job.go

```diff
@@ -107,6 +107,56 @@ func (c *TfJobSpec) Validate() error {
 	return nil
 }
 
+// ConfigureAccelerators adds any accelerator specific configuration to the pods.
+func (c *TfJobSpec) ConfigureAccelerators(accelerators map[string]AcceleratorConfig) error {
+	for _, r := range c.ReplicaSpecs {
+		if r.Template == nil {
+			return fmt.Errorf("Replica is missing Template; %v", util.Pformat(r))
+		}
+		for i, c := range r.Template.Spec.Containers {
+			if c.Name == string(TENSORFLOW) {
```

Comment:
I don't really understand why we specifically need to ensure that the container is named "tensorflow", but as this is not related to this PR we can discuss later.